wrangler.toml

```diff
@@ -51,13 +51,15 @@ command = "npm run build"
 [env.production.vars]
 MAX_SHARDS = "825"
 FF_RATE_LIMITER_ENABLED = "false"
-FF_EGRESS_TRACKER_ENABLED = "false"
+FF_EGRESS_TRACKER_ENABLED = "true"
 FF_TELEMETRY_ENABLED = "true"
+FF_DELEGATIONS_STORAGE_ENABLED = "true"
 GATEWAY_SERVICE_DID = "did:web:w3s.link"
 UPLOAD_SERVICE_DID = "did:web:web3.storage"
 CONTENT_CLAIMS_SERVICE_URL = "https://claims.web3.storage"
 CARPARK_PUBLIC_BUCKET_URL = "https://carpark-prod-0.r2.w3s.link"
 UPLOAD_API_URL = "https://up.web3.storage"
+INDEXING_SERVICE_URL = "https://indexer.storacha.network/"
```

Comment:
@alanshaw Could you confirm if this is the correct URL for the Production Indexer?

Reply:
👍